client/app/account/component/AccountInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { MdAccountCircle } from 'react-icons/md'
+import { colorMap } from '@/_consts/iconColor'
+import { UserType } from '@/_types'
+
+type AccountInfoProps = {
+  user: UserType
+}
```

Comment:
You actually don't have to pass user info as props.  When a user accesses this application(anywhere) initially, the user data is already fetched.  So you can just use `useGetMeQuery` in this component directly.

Reply:
Got it! Fixed it. 
f2f47aee4e897c82546571c8039808bb464940ee